packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -24,6 +24,8 @@ import Field from "../Forms/Field";
 import { getValidDate } from "../../services/dates";
 import { GBAddCircle } from "../Icons";
 import SelectField from "../Forms/SelectField";
+import ImportExperimentList from "./ImportExperimentList";
```

Comment:
I think we should take a different approach.

1.  Revert all changes in this `NewExperimentForm` component (already really hard to maintain)
2.  Make a new `ImportExperimentModal` component:
     ```tsx
     const [selected, setSelected] = useState<null|Partial<ExperimentInterfaceStringDates>>(null);

     if(selected) {
       return <NewExperimentForm initialValue={selected} .../>
     }

     return (
       <Modal ...>
         <ImportExperimentList
            ...
            onImport={(exp) => setSelected(exp)}
         />
       </Modal>
     )
     ```

Reply:
I can do this, but there would be no way to quickly switch between a new exp and an import. Do you think this is okay?